src/services/module_service.ts

```diff
@@ -3,6 +3,9 @@ import { colours, LoggerService } from "../../deps.ts";
 import DenoService from "../services/deno_service.ts";
 import NestService from "../services/nest_service.ts";
 
+const supportedUrls = ["https://deno.land/", "https://x.nest.land"];
+const importVersionRegex = /(v)?[0-9\.]+[0-9\.]+[0-9]/g;
```

Comment:
is it okay that we match 1.2.3.4.5.6.7.8.9.12.312.313.1.231.3.123.12.31.23.1.23.2 as a version number?

Reply:
Yeah - what's the reason?

Reply:
just wondering. didn't know if we wanted to only match v1.2.3 or 1.2.3 and not 1.2.3.1.12.312.3.12.31.23.1.233

Reply:
actually... we might have to look into beefing this regex up to support versions like v1.2.3-rc1

Reply:
ill make an issue about it